snuba/datasets/storages/errors.py

```diff
@@ -165,16 +164,18 @@
             write_schema=schema,
             read_schema=schema,
             required_columns=required_columns,
-            tag_column_map={
-                "tags": promoted_tag_columns,
-                "contexts": {},
-            },
-            promoted_tags={
-                "tags": promoted_tag_columns.keys(),
-                "contexts": {},
-            },
+            tag_column_map={"tags": promoted_tag_columns, "contexts": {},},
+            promoted_tags={"tags": promoted_tag_columns.keys(), "contexts": {},},
             state_name=ReplacerState.ERRORS,
         ),
     ),
-    query_processors=[PrewhereProcessor()],
+    query_processors=[
+        PostReplacementConsistencyEnforcer(
+            project_column="project_id",
+            # key migration is on going. As soon as all the keys we are interested
+            # into in redis are stored with "EVENTS" in the name, we can change this.
```

Comment:
I am confused about this comment.

To my recollection, this isn't a key migration — these two storages just have distinct replacement state, so I don't understand what "we can change this" means in this context. And shouldn't this be "ERRORS" and not "EVENTS" (since this is the `errors` storage?)

It looks like this was just copy/pasted from here but not updated? https://github.com/getsentry/snuba/blob/53e53f2ee12f6bfe03ae940c22c8932fa3c4c209/snuba/datasets/events.py#L98-L103

Reply:
I copied the comment by mistake, sorry. There should not be any comment here, as there is none in the errors dataset.